### DISCUSSION TITLE
warnings while using w64devkit to "make"

### DISCUSSION THREAD

**Discussion by NBcobo** [LostRuins/koboldcpp#137] [Q&A]
I tried to compile the exe file using the source code. I got these warnings when running "make" in w64devkit terminal. Is it ok to move on to the next step? I did see those .dll files created.
![image](https://user-images.githubusercontent.com/130814579/236083122-102813c2-414f-48cf-b81d-e6c4a24452e4.png)

**Comment by LostRuins** [comment]
Yes the RWKV warnings are harmless. Just leave them there. I'll probably hide them in future
